src/class_addressGenerator.py

```diff
@@ -143,8 +134,8 @@ def run(self):
                         potentialPubSigningKey + potentialPubEncryptionKey)
                     ripe = RIPEMD160Hash(sha.digest()).digest()
                     if (
-                        ripe[:numberOfNullBytesDemandedOnFrontOfRipeHash] ==
-                        '\x00' * numberOfNullBytesDemandedOnFrontOfRipeHash
+                        ripe[:numberOfNullBytesDemandedOnFrontOfRipeHash]
+                        == '\x00' * numberOfNullBytesDemandedOnFrontOfRipeHash
```

Comment:
b'\x00' because ripe is of type bytes

Reply:
In a later PR.

Reply:
That's another problem of such development: you are digging too shallow, and forcing other developers to do the same ): after each such purely formatting PR I forced to rebase two or three of my branches, containing much more nontrivial changes as well as most of such formatting fixes. 5 PRs per day is abnormal volume, moreover because the most of them are got abandoned.

Reply:
I'm trying to apply principles of continuous delivery and not have long lived branches. Then the whole process is easier: it takes less time to review, less time to rebase, and merging can happen more frequently.